src/components/form/TextInput/TextInput.js

```diff
@@ -0,0 +1,19 @@
+import { StyledTextInput } from "./TextInput.styled";
+import { useInputState } from "../../../services/hooks/useInputState";
+
+export const TextInput = ({ label, field, handleForm }) => {
+  const [text, handleText] = useInputState("");
```

Comment:
`value, setValue`, `inputValue, setInputValue` -> those are more readable for me and better if `TextInput` and `NumberInput` became just `Input` ;d

Reply:
but i think about it for few minutes and my conclusion is that components like custom `<Input /> should be more generic so we should have in component it owns state managament, rather we should pass value and onChange action